### PR TITLE
67 finish my orgs

### LINKED ISSUE #67: Finish My Orgs page
criteria:

- be able to refresh organizations
- include the names of boards within the organization
- route to boards correctly

### PR DESCRIPTION
**Pull request by tylerminnis**
Closes #67

Screen Shot 2023-11-15 at 02 03 22

### Progress Update

- orgs should be working almost 100% now, you gotta make sure u have orgs but it shouldn't crash if u don't. lmk if it does
- still need to get creating orgs working
- still need to update the link to individual boards
- actually ^ need to redo router
- 
other than that it works!

### CODE REVIEW

File: src/components/SpecificOrganization.tsx [hunk 1]
```diff
@@ -1,28 +1,69 @@
...
-                        listImg: "https://s3.us-east-1.wasabisys.com/sync-space/logo/SyncSpace-mint.png",
-                    }
-                ]} />
+            {loading ? <IonSpinner color="primary" className="ion-text-center ion-padding" />
+                : boardNames.length > 0 ? 
+                <CustomList
+                    title={org.name}
+                    titleImg="https://s3.us-east-1.wasabisys.com/sync-space/logo/SyncSpace-mint.png"
+                    items={boardNames}
+                /> : <p>nothing loaded</p>}
```
Comment (tylerminnis) [inline src/components/SpecificOrganization.tsx:66]: this really should be a better warning but not mvp

File: src/pages/MyOrgs.tsx [hunk 2]
```diff
@@ -103,6 +115,10 @@ const MyOrgs: React.FC = () => {
           </IonToolbar>
         </IonHeader>
         <IonSearchbar></IonSearchbar>
+        <IonRefresher slot="fixed" onIonRefresh={handleRefresh}>
+          <IonRefresherContent>
+          </IonRefresherContent>
+        </IonRefresher>
```
Comment (tylerminnis) [inline src/pages/MyOrgs.tsx:121]: this is fun bc u can pull down to refresh in case things don't load correctly

File: src/main.tsx [hunk 3]
```diff
@@ -15,8 +15,8 @@ const root = createRoot(container!);
 
 root.render(
   <Auth0Provider
-    domain={auth0Domain}
-    clientId={clientId}
+    domain={auth0Domain!}
```
Comment (tylerminnis) [inline src/main.tsx:18]: no more squiggly lines

**Review by sumiv — APPROVED** [review]
lets get this bread

**Pull request merged**